pkg/inject/proxy.go

```diff
@@ -43,6 +44,8 @@ func (m *proxyMutator) mutate(pod *corev1.Pod) error {
 }
 
 type proxyConfig struct {
+	// RunAsUser for init container
+	runAsUser int64
```

Comment:
Why is the init container config under proxy configs?

Reply:
I am not sure why it is there, but I only find this variable used in initProxyMutator and cniProxyMutator so it should not affect the envoy config.